### ISSUE TITLE
After initial `crc start -b xxx.crcbundle`, `crc start` without `-b` should work

### ISSUE THREAD

**Issue by cfergeau** [code-ready/crc#952]
At the moment, it looks for `~/.crc/cache/xxx.crcbundle` and fail since it's missing,
which is not going to be a problem as
- the bundle is uncompressed in `~/.crc/cache/xxx`
- `~/.crc/machines/crc` already exists, so we don't need the data from `~/.crc/cache`

**Comment by praveenkumar** [comment]
@cfergeau yes, I think we did discuss this, have to find out where was that discussion :)

**Comment by zhangli19817** [comment]
$ crc start
ERRO file '/Users/luffy/.crc/cache/crc_hyperkit_4.2.13.crcbundle' does not exist 
ERRO Please provide the path to a valid bundle using the -b option 

Question:

where's the target crc_hyperkit_4.2.13.crcbundle?
on this mac platform, with flag -b, what's this parameter refer to?

**Comment by cfergeau** [comment]
If you use a released version from `mirror.openshift.com`, `crc setup` will unpack the needed bundle to `~/.crc/cache`
